### DISCUSSION TITLE
I can't start my server with mods

### DISCUSSION THREAD

**Discussion by RubenLopezP** [IzzelAliz/Arclight#809] [Q&A]
I try to start the server with any mod and I get this error that I don't know how to solve.

[crash-2022-11-08_14.05.49-server.txt](https://github.com/IzzelAliz/Arclight/files/9961488/crash-2022-11-08_14.05.49-server.txt)

**Comment by PSYCHOFOX81** [comment]
Make sure the mod is 1.18.2 and make sure your using the correct forge version!

https://www.curseforge.com/minecraft/mc-mods/security-craft/files/3921270
